pandas/io/json.py

```diff
@@ -744,6 +744,9 @@ def json_normalize(data, record_path=None, meta=None,
         If True, prefix records with dotted (?) path, e.g. foo.bar.field if
         path to records is ['foo', 'bar']
     meta_prefix : string, default None
+    separator : string, default '.'
```

Comment:
Can you add a [`.. versionadded::`](http://www.sphinx-doc.org/en/1.4.9/markup/para.html#directive-versionadded) directive here.

Also, might be better to make `separator` the last keyword argument. That way it won't break people using all positional arguments.